File: go/leap/leap.go
package leap

const testVersion = 2

// IsLeapYear returns true if year is a leap year
func IsLeapYear(year int) bool {
	if year % 4 == 0 {
		if year % 400 == 0 {
			return true
		} else if year % 100 == 0 {
			return false
		} else {
			return true
		}
	}
	return false
}

File: python/leap/leap.py
"""Module for testing for leap years"""


def is_leap_year(year):
    """"is_leap_year returns true if provided year is a leap year and false otherwise."""
    if year % 400 == 0:
        return True
    elif year % 100 == 0:
        return False
    elif year % 4 == 0:
        return True
    return False